### PR TITLE
Update BAR extensions

### PR DESCRIPTION
**Pull request by Xeeynamo**
Most of them are official extensions. This is still a bit work in progress as we still miss few information (eg. extensions for `MON` and `PLX`)

### CODE REVIEW

**Comment by kenjiuno** [comment]
This is significant changes!

- `.ai` → `.bdx`
- `.doct` → `.ocd`
- `.coct` → `.occ`
- `.modeltexture`　→ `.texture`
- `.animation` → `.motion`
- `.texture` → `.tm2`
- `.coctcamera` → `.och`
- `.colordiffuse`　→ `.fog`
- `.coctlight` → `.ocl`
- `.Anb` → `.triggers` (Anb → MotionTriggers)
- `.MapCollision2` → `.owa`
- `.bobdesc` → `.bop` (BobDescriptor → BgObjPlacement)
- `.sed` → `.sqd`

File: OpenKh.Kh2/Bar.cs [hunk 1]
```diff
@@ -49,25 +49,25 @@ public enum EntryType
...
+			Motion = 9,
+			Tim2 = 10,
             CameraCollision = 11,
             SpawnPoint = 12,
             SpawnScript = 13,
-            MapColorDiffuse = 14,
+            FogColor = 14,
             LightData = 15,
-            Anb = 16,
-			Bar = 17,
```
Comment (kenjiuno) [inline OpenKh.Kh2/Bar.cs:66]: I think this Bar → Anb change is unintentional.

Comment (kenjiuno) [reply OpenKh.Kh2/Bar.cs:66]: Sorry this will be intentional.

**Review by kenjiuno — APPROVED** [review]

**Comment by Xeeynamo** [comment]
> This is significant changes!
> 
> * `.ai` → `.bdx`
> * `.doct` → `.ocd`
> * `.coct` → `.occ`
> * `.modeltexture`　→ `.texture`
> * `.animation` → `.motion`
> * `.texture` → `.tm2`
> * `.coctcamera` → `.och`
> * `.colordiffuse`　→ `.fog`
> * `.coctlight` → `.ocl`
> * `.Anb` → `.triggers` (Anb → MotionTriggers)
> * `.MapCollision2` → `.owa`
> * `.bobdesc` → `.bop` (BobDescriptor → BgObjPlacement)
> * `.sed` → `.sqd`

Those are the official file name extensions :) we recently discover them.

**Pull request merged**